src/main/java/org/springframework/data/couchbase/config/AbstractCouchbaseConfiguration.java

```diff
@@ -145,14 +146,14 @@ protected void configureEnvironment(final ClusterEnvironment.Builder builder) {
 
 	@Bean(name = BeanNames.COUCHBASE_TEMPLATE)
 	public CouchbaseTemplate couchbaseTemplate(CouchbaseClientFactory couchbaseClientFactory,
-			MappingCouchbaseConverter mappingCouchbaseConverter) {
-		return new CouchbaseTemplate(couchbaseClientFactory, mappingCouchbaseConverter);
+			MappingCouchbaseConverter mappingCouchbaseConverter, TranslationService translationService) {
```

Comment:
The translationService argument should be named couchbaseTranslationService to match the bean name.
When the argument name does not match the bean name, the spring framework can only match by the type, and if there is ever another bean of type TranslationService added, the spring framework will not be able to automatically match this argument with the couchbaseTranslationService bean.